### PR TITLE
add request parameters for postCreateTask

### PR DESCRIPTION
**Pull request by valentin-chaplean**
This change is [Reviewable](https://reviewable.io/reviews/chaplean/kanbanize-api-client-bundle/4)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,5 +28,17 @@ New features:
         * Get a list of successor cards
         * Make a card a successor of a given card 
         
+Bug fixes:
+    **None**
+
+## v1.0.1
```
Comment (mdevlamynck) [inline CHANGELOG.md]: Newer releases should be at the top of the file.

**Review by mdevlamynck — CHANGES_REQUESTED** [review]

File: Tests/Api/KanbanizeApiV1Test.php [hunk 2]
```diff
@@ -332,6 +332,25 @@ public function testPostCreateNewTaskWithExtraFields(): void
...
+     * @return void
+     */
+    public function testPostCreateNewTaskWithWorkflowField(): void
+    {
+        $response = $this->api->postCreateTask()
+            ->bindRequestParameters([
+                'boardid'      => 1,
+                'workflow'     => 1,
+                'workflowid'   => 1,
+                'workflowname' => 'Intiatives cards',
```
Comment (tomchaplean) [inline Tests/Api/KanbanizeApiV1Test.php]: Typo: Initiatives

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v1.0.1
```
Comment (tomchaplean) [inline CHANGELOG.md]: Not a bugfix version but new features one, so it should be 1.1.0 (cf Versioning in README)

**Review by tomchaplean — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (valentin-chaplean) [reply CHANGELOG.md]: Oops indeed ^^

**Review by tomchaplean — APPROVED** [review]

**Review by mdevlamynck — APPROVED** [review]

**Pull request merged**